lib/manager/components/version/FeedVersionDetails.js

```diff
@@ -111,6 +111,28 @@ export default class FeedVersionDetails extends Component<Props> {
     const mergeButtonLabel = isMergedServicePeriods
       ? 'Cannot re-merge feed'
       : 'Merge with version'
+
+    const hasMtcExtension = isExtensionEnabled('mtc')
+    let publishWarningMessage
+    if (hasMtcExtension) {
+      if (hasBlockingIssue || hasGtfsPlusBlockingIssue) {
+        publishWarningMessage = (
+          <span>
+            Cannot publish version because it has a{' '}
+            {hasGtfsPlusBlockingIssue ? 'GTFS+ ' : ''}
+            blocking issue.
```

Comment:
Is it worth saying something else if there is a GTFS+ issue as well as a non-GTFS+ issue?

Reply:
I think this text was meant to guide the user either to GTFS+ issues or standard validation issues if any.